setup/crm_automated_activity/odoo/addons/crm_automated_activity

```diff
@@ -0,0 +1 @@
+../../../../crm_automated_activity
```

Comment:
As far as I know, you don't have to add this manually either.

Reply:
Ok